### PR TITLE
Added recipe for jupyter_qtconsole_colorschemes

### PR DESCRIPTION
**Pull request by pmlandwehr**
A collection of color schemes for `jupyter qtconsole`

### CODE REVIEW

**Comment by pmlandwehr** [comment]
If anyone in in @conda-forge/staged-recipes has a chance to review this dumb thing, please do.

File: recipes/jupyter_qtconsole_colorschemes/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
```
Comment (synapticarbors) [inline recipes/jupyter_qtconsole_colorschemes/meta.yaml]: It looks like `pygments` is required. Also is `setuptools` a runtime requirement?

Comment (synapticarbors) [inline recipes/jupyter_qtconsole_colorschemes/meta.yaml]: nevermind, doing a quick search, I realized that the `.py` files are pygments files so they are more like resources than importable modules.

File: recipes/jupyter_qtconsole_colorschemes/LICENSE [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Copyright (C) 2014 Joon Ro
```
Comment (jakirkham) [inline recipes/jupyter_qtconsole_colorschemes/LICENSE]: Would raise an issue with upstream about getting this included in the `MANIFEST.in`. Seems like this is their [issue tracker]( https://bitbucket.org/joon/color-schemes-for-jupyter-qt-console/issues?status=new&status=open ).

**Pull request merged**